src/psyclone/tests/psyir/transformations/kernel_transformation_test.py

```diff
@@ -430,6 +431,24 @@ def test_gpumixin_validate_no_call():
             in str(err.value))
 
 
+@pytest.mark.parametrize(
+    "rtrans, expected_directive",
+    [(ACCRoutineTrans(), "!$acc routine"),
+     (OMPDeclareTargetTrans(), "!$omp declare target")])
+def test_kernel_gpu_annotation_trans(rtrans, expected_directive,
+                                     fortran_writer):
+    ''' Check that the kernel GPU annotation transformations insert the
+    porper directive into the kernel '''
```

Comment:
"proper"

Reply:
also, worth saying "PSyKAl kernel" as I missed the significance of this test until I got to the code changes.

Reply:
Done